### PR TITLE
[RAPTOR-3643] import keras from TF in drum keras predictor

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Add a check for keras model is instance of keras included into tensorflow.

### CODE REVIEW

**Review by 4dahalibut — APPROVED** [review]

File: custom_model_runner/datarobot_drum/drum/artifact_predictors/keras_predictor.py [hunk 1]
```diff
@@ -47,12 +47,13 @@ def can_use_model(self, model):
         try:
             import keras
             from sklearn.pipeline import Pipeline
+            from tensorflow import keras as keras_tf
 
             if isinstance(model, Pipeline):
                 # check the final estimator in the pipeline is Keras
-                if isinstance(model[-1], keras.Model):
+                if isinstance(model[-1], (keras.Model, keras_tf.Model)):
```
Comment (4dahalibut) [inline custom_model_runner/datarobot_drum/drum/artifact_predictors/keras_predictor.py:54]: Is there any parent class they both depend on?

Comment (yakov-g) [reply custom_model_runner/datarobot_drum/drum/artifact_predictors/keras_predictor.py:54]: No, looks like there is no.
As I understand, Keras is API standard which can use different backends.

Starting tensorflow 2 - tensorflow implements keras.
https://blog.tensorflow.org/2018/12/standardizing-on-keras-guidance.html

So technically, tensorflow >= 2 doesn't require keras installation.

Comment (meltzg) [reply custom_model_runner/datarobot_drum/drum/artifact_predictors/keras_predictor.py:54]: That depends on the version of keras.  In the latest version of standalone keras, they're the same class. In the version included with the public env, the only common parent is object

**Review by meltzg — APPROVED** [review]

File: custom_model_runner/datarobot_drum/drum/artifact_predictors/keras_predictor.py (hunk 1 above)
Comment (yakov-g) [reply custom_model_runner/datarobot_drum/drum/artifact_predictors/keras_predictor.py:54]: I see, other backends has been discontinued: https://github.com/keras-team/keras/blob/master/README.md

**Comment by datarobot-jarvis** [comment]
:dollar: Total cost of executing tests in Jenkins: $**1.23** USD.

[What is this all about?](http://bit.ly/2BahK4V)

**Pull request merged**